samples/client/petstore/python-experimental/petstore_api/model_utils.py

```diff
@@ -883,6 +883,46 @@ def deserialize_primitive(data, klass, path_to_item):
         )
 
 
+def get_discriminator_class(model_class, model_data, cls_visited):
+    """Returns the child class specified by the discriminator.
+
+    Args:
+        model_class (OpenApiModel): the model class
+        model_data (list/dict): data to instantiate the model.
+        cls_visited (set): list of model classes that have been visited.
```

Comment:
cls_visited should be a tuple, not a set. Our classes are not hashable so we need to make sure to store them in a tuple not a set.